recipes/btyacc/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import functools
+import os
+
+from conans import CMake, ConanFile, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class BisonConan(ConanFile):
+    name = "btyacc"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ChrisDodd/btyacc"
+    description = "Backtracking yacc"
+    topics = ("yacc", "parser")
+    license = "???"
```

Comment:
README says freeware, README.BYACC says public domain. What should go in here?

Reply:
Relevant response from the author: https://github.com/ChrisDodd/btyacc/issues/28#issuecomment-1014894685

Reply:
if it's public domain, then it's `Unlicense` SPDX identifier

Reply:
```suggestion
    license = "Unlicense"
```